src/services/migration/HomeAuthXmlMigration.php

```diff
@@ -36,7 +36,7 @@ public function migrate(): void {
         $newContent = \str_replace('https://phar.io/phive-auth', 'https://phar.io/auth', $oldContent);
         $new->putContent($newContent);
 
-        $this->output->writeText('Migration of global authentication configuration almost finish.');
+        $this->output->writeText('Migration of global authentication configuration almost finish. ');
```

Comment:
Two problems:

- English grammer: It probably should be "finished"
- I'm not sure the sentence is useful as is. Maybe it should be merged with the question below?